chrome/model.js

```diff
@@ -385,6 +454,10 @@ GmailToTrello.Model.prototype.Uploader.prototype = {
 
             let self = this;
             fn_k(method, property, upload1, function success(data) {
+                if (data.idBoard) {
+                  self.parent.trello.threadCards.push(data); // if it's a card, add to the list
+                  self.parent.sortThreadCards();
+                }
```

Comment:
Every time you upload, if you uploaded a card (rather than an attachment), take the card data and add it to threadCards so it will display in the button and when you re-open the popup.